services/gossip/adapter/file/file_topology.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2020 the orbs-network-go authors
+// This file is part of the orbs-network-go library in the Orbs project.
+//
+// This source code is licensed under the MIT license found in the LICENSE file in the root directory of this source tree.
+// The above notice should be included in all copies or substantial portions of the software.
+
+package file
```

Comment:
suggestion - add two things to this implementation - 
- validation for the `ip` string and the range of `port`
- protection against very large files. use a configured value with a default of 1MB for instance

Reply:
done